homeassistant/components/axis/.translations/en.json

```diff
@@ -0,0 +1,48 @@
+{
+    "config": {
+        "title": "Axis device",
+        "step": {
+            "user": {
+                "title": "Set up Axis device",
+                "data": {
+                    "host": "Host",
+                    "username": "Username",
+                    "password": "Password",
+                    "port": "Port"
+                }
+            },
+            "features": {
+                "title": "Select features from Axis device",
+                "data": {
+                    "camera": "MJPEG video stream",
+                    "daynight": "Day/Night mode",
+                    "input": "Input port",
+                    "motion": "Motion detection",
+                    "pir": "PIR sensor",
+                    "sound": "Sound detector",
+                    "tampering": "Tampering sensor",
+                    "vmd3": "Video Motion Detection 3",
+                    "vmd4": "Video Motion Detection 4"
+                }
+            },
+            "options": {
```

Comment:
Same, options are going to be added later.

Reply:
If you want to keep allowing people to configure this, make it happen via `configuration.yaml`.

Reply:
@balloob I would much rather keep on this path (Im doing a similar thing with a couple of deconz options) until the real options functionality is available. I have no issues refactor the config flow as that is available. I really just want to get rid of configuration.yaml since the devices supported are so dynamic in what they support.